src/fable-library/Fable.Library.fsproj

```diff
@@ -8,9 +8,9 @@
 
   <ItemGroup>
     <Compile Include="Global.fs" />
-    <Compile Include="BigInt/n.fsi" />
+    <!-- <Compile Include="BigInt/n.fsi" /> -->
```

Comment:
@alfonsogarciacaro Not sure why those are commented, .fsi files define proper visibility and access.